step-templates/sql-execute-sql-agent-job.json

```diff
@@ -5,7 +5,7 @@
   "ActionType": "Octopus.Script",
   "Version": 13,
```

Comment:
`Version` needs to be bump, otherwise the step template update won't show up in Octopus.